### ISSUE TITLE
AttributeError: module 'deepface.DeepFace' has no attribute 'extract_faces'

### ISSUE THREAD

**Issue by anu1097** [serengil/deepface#771]
I ensured and double checked pip's installed library is latest. 

`pip show deepface`              

```
Name: deepface
Version: 0.0.79
Summary: A Lightweight Face Recognition and Facial Attribute Analysis Framework (Age, Gender, Emotion, Race) for Python
Home-page: https://github.com/serengil/deepface
Author: Sefik Ilkin Serengil
Author-email: serengil@gmail.com
License: UNKNOWN
Location: /opt/homebrew/lib/python3.10/site-packages
Requires: fire, Flask, gdown, gunicorn, keras, mtcnn, numpy, opencv-python, pandas, Pillow, retina-face, tensorflow, tqdm
Required-by: 
```

I checked inside the code in my modules repository. extract_faces is missing. Is the latest library present in pypi ?
I am facing issues due to this. On Github extract_faces is present.

**Comment by serengil** [comment]
yes this is available in the latest pip package: https://files.pythonhosted.org/packages/14/40/89554f19133bdab441b1baabf0c4942b022277f4470a0df501eadbd93570/deepface-0.0.79.tar.gz

**Comment by anu1097** [comment]
Brother I have tried everything. Uninstalled the module from pip. Tried to install it using conda didn't work. 

Cloned it from the repo and tried to install from there too. Still this function isn't present. Is downloading the directory you shared and putting it in site-packages only solution ?

**Comment by serengil** [comment]
most probably you are using a different virtual environment.

does not matter using the latest pip package or source code. they both offer this function.

**Comment by anu1097** [comment]
I think I am going to uninstall conda. It has caused ton of problems to me.
